common/data/segment-values.ts

```diff
@@ -55,8 +55,10 @@ export type ItemLinkSource =
   | 'viewer/scroll'
   | 'viewer/resize'
   | 'manifests_navigation'
-  | 'search_within_result'
-  | 'contents_nav';
+  | 'contents_nav'
+  | 'search_within_submit'
+  | 'search_within_clear'
```

Comment:
I'm not sure how sustainable this is as a process but since I've just added [this table of possible sources in GitBook](https://app.gitbook.com/o/-LumfFcEMKx4gYXKAZTQ/s/MM8zK9GZyFCpy8kvd5Km/analytics-and-reporting/analytics-event-tracking#segment-events), should we add the new one as well?